manage-data/lifecycle/data-tiers.md

```diff
@@ -2,31 +2,447 @@
+
+The frozen tier requires a snapshot repository. The frozen tier uses [partially mounted indices](/deploy-manage/tools/snapshot-and-restore/searchable-snapshots.md#partially-mounted) to store and load data from a snapshot repository. This reduces local storage and operating costs while still letting you search frozen data. Because {{es}} must sometimes fetch frozen data from the snapshot repository, searches on the frozen tier are typically slower than on the cold tier.
+
+## Configure data tiers [configure-data-tiers]
+
+Follow the instructions for your deployment type to configure data tiers.
+
+### {{ess}} or {{ece}} [configure-data-tiers-cloud]
+
+The default configuration for an {{ecloud}} deployment includes a shared tier for hot and content data. This tier is required and can’t be removed.
+
+#### Add a data tier
+
+To add a warm, cold, or frozen tier when you create a deployment:
+
+1. On the **Create deployment** page, click **Advanced Settings**.
+2. Click **+ Add capacity** for any data tiers to add.
+3. Click **Create deployment** at the bottom of the page to save your changes.
+
+:::{image} ../../images/elasticsearch-reference-ess-advanced-config-data-tiers.png
+:alt: {{ecloud}}'s deployment Advanced configuration page
+:class: screenshot
+:::
+
+To add a data tier to an existing deployment:
+
+1. Log in to the [{{ecloud}} console](https://cloud.elastic.co?page=docs&placement=docs-body).
+2. On the **Deployments** page, select your deployment.
+3. In your deployment menu, select **Edit**.
+4. Click **+ Add capacity** for any data tiers to add.
+5. Click **Save** at the bottom of the page to save your changes.
+
+#### Disable a data tier [disable-a-data-tier]
+
+The process of disabling a data tier depends on whether we are dealing with [searchable snapshots](#ece-disable-searchable-snapshot-data-tier) or [regular indices](#ece-disable-non-searchable-snapshot-data-tier).
+
+The hot and warm tiers store regular indices, while the frozen tier stores searchable snapshots. However, the cold tier can store either regular indices or searchable snapshots. To check if a cold tier contains searchable snapshots perform the following request:
+
+```sh
+GET /_cat/indices/restored-*
+```
+
+##### Non-searchable snapshot data tier [ece-disable-non-searchable-snapshot-data-tier]
+
+Elastic Cloud Enterprise tries to move all data from the nodes that are removed during plan changes. To disable a non-searchable snapshot data tier (e.g., hot, warm, or cold tier), make sure that all data on that tier can be re-allocated by reconfiguring the relevant shard allocation filters. You’ll also need to temporarily stop your index lifecycle management (ILM) policies to prevent new indices from being moved to the data tier you want to disable.
+
+To learn more about ILM for Elastic Cloud Enterprise, or shard allocation filtering, check the following documentation:
```

Comment:
```suggestion
To learn more about ILM for {{ece}}, or shard allocation filtering, check the following documentation:
```
I suppose we may fix all these entities automagically at some point, so feel free to ignore.